modules/core/include/opencv2/core/hal/intrin_rvv.hpp

```diff
@@ -584,63 +587,84 @@ OPENCV_HAL_IMPL_RVV_SELF_REINTERPRET(int64x2, s64)
 OPENCV_HAL_IMPL_RVV_SELF_REINTERPRET(float64x2, f64)
 #endif
 
-#define OPENCV_HAL_IMPL_RVV_ONE_TIME_REINTERPRET(_Tpvec1, _Tpvec2, _nTpvec1, _nTpvec2, suffix1, suffix2, nsuffix1, nsuffix2, width1, width2, vl1, vl2) \
+#define OPENCV_HAL_IMPL_RVV_NATIVE_REINTERPRET(_Tpvec1, _Tp1, _Tpvec2, _Tp2, _nTpvec1, _nTpvec2, suffix1, suffix2, nsuffix1, nsuffix2, width1, width2, vl1, vl2) \
```

Comment:
width1, width2, vl1, vl2 are not used any more. I propose to simplify the macro and it's usage.

Reply:
Agree.
`_Tp1`, `_Tpvec2`, `_Tp2`, `_nTpvec1`, `width1`, `width2`, `vl1`, `vl2` in NATIVE_REINTERPRET and 
 `_Tp1`, `_Tpvec2`, `_Tp2`, `_nTpvec1`, `vl1`, `vl2` in TWO_TIMES_REINTERPRET are not used.
Removed.